frontend/src/components/Error.tsx

```diff
@@ -0,0 +1,18 @@
+import {Link} from "react-router-dom";
+
+export default function Error({message, link}: {
+    message: string,
+    link?: {
+        text: string,
+        to: string
+    }
+}) {
+    return (
+      <div className="message-container">
```

Comment:
Is the name of this container a bit too ambiguous here? Perhaps error-message-container?

Reply:
guter punkt - is geändert